doc/manual/cookbooks/burnman/plot_reference_profile.plt

```diff
@@ -0,0 +1,55 @@
+# Plot the reference state
+
+set ylabel "Depth in km"
+set xlabel "Temperature in K"
+unset key 
+set yrange [2800:0.0]
+set datafile missing "nan"
+set xtics (1600, 2000, 2400)
+
+set terminal postscript color portrait dashed enhanced 'Arial' size 40cm,15cm
+set output 'reference_profile.eps'
+
+set multiplot layout 1,6
+
+set bmargin at screen 0.15
+set lmargin 9
+set rmargin 1
+
+set title "Reference temperature"
+plot "../../../../data/adiabatic-conditions/ascii-data/isentrope_properties.txt" using 3:($1/1000) with lines linecolor rgb "gray" lw 4 lt 1 title "temperature"
```

Comment:
That would have been due to the latent heat, but we decided to run the model without latent heat effects (and the updated profile does not have the drop any more).